### PR TITLE
[I18N][15.0] stock: Add translation

### PR DESCRIPTION
**Pull request by ntlquynh2209**
Description of the issue/feature this PR addresses:
https://viindoo.com/web#id=8779&menu_id=89&cids=1&model=helpdesk.ticket&view_type=form

Current behavior before PR:
Miss translation

Desired behavior after PR is merged:
Add translation

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/i18n/vi.po [hunk 1]
```diff
@@ -3126,7 +3126,7 @@ msgstr ""
 #. module: stock
 #: model:ir.model.fields.selection,name:stock.selection__stock_storage_category__allow_new_product__same
 msgid "If all products are same"
-msgstr ""
+msgstr "Cho phép sản phẩm cùng loại"
```
Comment (vietphuong97) [inline addons/stock/i18n/vi.po]:
```suggestion
msgstr "Nếu tất cả sản phẩm giống nhau"
```

File: addons/stock/i18n/vi.po [hunk 2]
```diff
@@ -3181,7 +3181,7 @@ msgstr ""
 #. module: stock
 #: model:ir.model.fields.selection,name:stock.selection__stock_storage_category__allow_new_product__empty
 msgid "If the location is empty"
-msgstr ""
+msgstr "Cho phép nếu địa điểm trống"
```
Comment (vietphuong97) [inline addons/stock/i18n/vi.po]:
```suggestion
msgstr "Nếu địa điểm là trống"
```

File: addons/stock/i18n/vi.po [hunk 3]
```diff
@@ -7317,7 +7317,7 @@ msgstr ""
 #: model_terms:ir.ui.view,arch_db:stock.res_config_settings_view_form
 #: model_terms:ir.ui.view,arch_db:stock.stock_storage_category_tree
 msgid "Storage Categories"
-msgstr ""
+msgstr "Quy tắc lưu trữ"
```
Comment (vietphuong97) [inline addons/stock/i18n/vi.po]:
```suggestion
msgstr "Danh mục lưu trữ"
```

File: addons/stock/i18n/vi.po [hunk 4]
```diff
@@ -7337,7 +7337,7 @@ msgstr ""
 #: model:ir.ui.menu,name:stock.menu_storage_categoty_capacity_config
 #: model_terms:ir.ui.view,arch_db:stock.stock_storage_category_capacity_tree
 msgid "Storage Category Capacity"
-msgstr ""
+msgstr "Giới hạn lưu trữ"
```
Comment (vietphuong97) [inline addons/stock/i18n/vi.po]:
```suggestion
msgstr "Sức chứa Danh mục lưu trữ"
```

Comment (vietphuong97) [reply addons/stock/i18n/vi.po]: dùng theo thuật ngữ bên trên cho đồng bộ.

**Comment by vietphuong97** [comment]
Sửa lại theo phần anh review nhé @ntlquynh2209

**Comment by ntlquynh2209** [comment]
> Sửa lại theo phần anh review nhé @ntlquynh2209

@vietphuong10  em sửa rồi ạ

**Review by vietphuong10 — APPROVED** [review]

**Pull request merged**